WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -29,30 +29,47 @@ class PluginViewModel: Observable {
     var dismiss: (() -> Void)?
 
     var tableViewModel: ImmuTable {
+
+        var header: ImmuTableRow?
+        if let directory = plugin.directoryEntry {
+            header = PluginHeaderRow(
+                directoryEntry: directory,
+                onLinkTap: { [unowned self] in
+                    guard let url = directory.authorURL else { return }
+                    self.presentBrowser(for: url)
+            })
+        }
+
         var versionRow: ImmuTableRow?
-        if let version = plugin.state.version {
-            versionRow = TextRow(
-                title: NSLocalizedString("Plugin version", comment: "Version of an installed plugin"),
-                value: version)
-        }
-
-        var availableUpdateRow: ImmuTableRow?
-        switch plugin.state.updateState {
-        case .updated:
-            break
-        case .available(let version):
-            let message = String(format: NSLocalizedString("Version %@ is available", comment: "Message to show when a new plugin version is available"), version)
-            availableUpdateRow = TextWithButtonRow(
-                title: message,
-                actionLabel: NSLocalizedString("Update", comment: "Button label to update a plugin"),
-                action: { [unowned self] (_) in
-                    ActionDispatcher.dispatch(PluginAction.update(id: self.plugin.id, site: self.site))
-                }
+        switch (plugin.state.version, plugin.state.updateState) {
+            case (let version?, .updated):
+                versionRow = TextRow(
+                    title: NSLocalizedString("Version \(version)", comment: "Version of an installed plugin"),
+                    value: NSLocalizedString("Installed", comment: "Indicates the state of the plugin")
+                )
+            case (let version?, .available(let newVersion)):
+                let message = String(format: NSLocalizedString("Version %@ is available", comment: "Message to show when a new plugin version is available"), newVersion)
+                let subtitle = String(format: NSLocalizedString("Version %@ installed", comment: "Message to show what version is currently installed when a new plugin version is available"), version)
+
+                versionRow = TextWithButtonRow(
+                    title: message,
+                    subtitle: subtitle,
+                    actionLabel: NSLocalizedString("Update", comment: "Button label to update a plugin"),
+                    action: { [unowned self] (_) in
```

Comment:
I think this is the same behavior that before, but now that the "Update" looks like a button and not just a label on the cell, it feels odd that tapping anywhere else in the row will trigger an update.

Reply:
Agree, changed the behavior.